egs/aishell2/s5/local/chain/tuning/run_tdnn_1d.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# _1d is as _1c, but with dropout schedule added, referenced from wsj
+# _1d is as _1a, but with i-vector and dropout schedule added, referenced from wsj
```

Comment:
OK, I am assuming this is not the entire difference because this one has pitch too.